app/Http/Controllers/StatusController.php

```diff
@@ -174,4 +175,19 @@ public function confirmationEmail()
       $email = new Email();
       $email->sendEmail('submitted', 'applicant', Auth::user()->email);
   }
+
+  /**
+   * When an app is submitted, automatically rates as 'no' if the GPA is too low
+   */
+  public function checkGPA($application)
```

Comment:
like have it as a function on the model? I think we'd be able to do everything from there. I had this somewhere else before too, the choices are overwhelming, it's like going to Essen. But I think inside the model makes sense. Then we would still call it like it is now in the Status Controller. Sound good?